docs/fast_data/configuration/config_maps/kafka_projection_updates.md

```diff
@@ -35,7 +35,6 @@ The Kafka Projection Updates is made of the following fields:
 
 There will be different behaviors based on the strategy value:
 - `__automatic__`: The automatic strategy handles the generation of the projection changes identifier automatically. This is the preferred method and in most cases it will be enough for your Strategy executions.
-- `__lookup__`: The lookup Strategy means the Projection Updates will be handled by the [Single View Patch](/fast_data/configuration/single_views.md#single-view-patch) method. So before using this value make sure you have correctly configured all the parts for the Single View Patch architecture in your project.
```

Comment:
are we deprecating lookup strategies? do you know if there will be any impact on customers?

Reply:
we are not deprecating, we're completely removing them. The lookup "strategies" are actually fake strategies as they do nothing but cast the pr-update event into a fake sv-trigger event that will be sent to the SVC. The thing is the svc also accepts normal pr-update messages since the beginning of the sv-patch implementation so there was no actual need for the SVTG to have this lookup strategies. We asked around and nobody is using the SVTG for that end, whoever uses the sv-patch sends directly pr-updates from the RTU to the SVC. With @danibix95 we decided that since we will be doing a breaking change anyway in the SVTG service we could also remove this to have a cleaner code

Reply:
Also, if someone is using the SVTG leveraging the lookup feature it means that, to adapt to the new version of the SVTG they will need to enable pr-updates in the RTU and configure the SVC to read from them. So the solution is pretty easy. If you prefer we could state that in the Changelog of the service, as sort of a migration guide

Reply:
if we are sure no one is using it it's ok like this